tests/shell/check_bashisms.sh

```diff
@@ -17,6 +17,7 @@ find -version > /dev/null 2>&1 > /dev/null && FIND='find scripts -regextype egre
 scripts=$($FIND -type f -not -regex \
 	  '.+(check-env-dep|gitignore|kdb_zsh_completion|run_dev_env|sed|(Docker|Jenkins|Vagrant)file.*|\.(cmake|fish|in|md|txt))$' | \
 	  xargs)
+exit_if_fail 'Unable to locate shell scripts via `find`'
```

Comment:
If we let it fail this way, we need to make sure that the script is not installed.

And maybe you want to take that chance and change the cascading ifs to ifs with AND so that CMake formatting works.